released/uc_sap_build_apps/README.md

```diff
@@ -0,0 +1,14 @@
+# Setting up SAP Build Apps
+
+To run this use case reliably, you run it in two steps.
+
+## Step 1 - Prepare subaccount and SAP Build Apps
+
+- Change to the folder `step1` and run the terraform script (`terraform init` and `terraform apply`).
+- Note down the extracted output variables
```

Comment:
A hint concerning Terraform output might be useful here.

Reply:
Makes sense.